perfkitbenchmarker/configs/default_config_constants.yaml

```diff
@@ -50,6 +55,10 @@ default_500_gb: &default_500_gb
     disk_type: standard
     disk_size: 500
     mount_point: /scratch
+  CloudStack:
+    disk_type: standard
```

Comment:
It doesn't look like CloudStackDisk respects the "disk_type" which gets passed in - if there's no way for it to use that type, then there's no need to specify it here.